sw/host/provisioning/ft_lib/src/lib.rs

```diff
@@ -186,9 +225,163 @@ fn send_rma_unlock_token_hash(
     Ok(())
 }
 
-const CERT_LABELS: [&str; 8] = [
-    "UDS", "CDI_0", "CDI_1", "TPM_EK", "CEK", "CIK", "Unknown", "Unknown",
-];
+// Helper functions used to pack LTV object and Certificate payload headers.
+fn make_obj_header(size: usize, otype: ObjType) -> Result<ObjHeaderType> {
```

Comment:
maybe we could move these to the ujson_lib: https://cs.opensource.google/opentitan/opentitan/+/master:sw/host/provisioning/ujson_lib/src/lib.rs ? since they are functions used to format data to read/write from/to the ujson LTV payload?

Reply:
I can do it if you feel strongly about it, but this  does not seem a particularly good idea to me: so far all users of these structures and functions are in this file, moving these to the ujson lib will require a lot of exports/imports.